### PR TITLE
Sort by columns

### PR DESCRIPTION
**Pull request by ckousik**
click on colums to sort
	modified:   task-graph-inspector/taskgraphinspector.jsx

### CODE REVIEW

File: task-graph-inspector/taskgraphinspector.jsx [hunk 1]
```diff
@@ -49,10 +49,16 @@ var TaskGraphInspector = React.createClass({
...
       taskGraphError:     undefined,
       taskGraph:          null,
-      taskGraphIdInput:   ''
+      taskGraphIdInput:   '',
       // Remark we'll cache task status structures under
       // 'task/<taskId>/status' as arrive from messages or when we load a
       // a task...
+
+      //used for sorting
+      order:{
```
Comment (eliperelman) [inline task-graph-inspector/taskgraphinspector.jsx]: Some whitespace formatting:

```js
order: {
  key: '',
  asc: false
}
```

File: task-graph-inspector/taskgraphinspector.jsx [hunk 2]
```diff
@@ -490,9 +496,30 @@ var TaskGraphInspector = React.createClass({
...
+
+  /** Sort by columns
+  TaskId
+  Name
+  State
+  Satisfied
+  Reruns
+  Relation
+  */
+  sortBy: function(col){
```
Comment (eliperelman) [inline task-graph-inspector/taskgraphinspector.jsx]: nit: whitespace: `sortBy: function(col) {`

Comment (jonasfj) [inline task-graph-inspector/taskgraphinspector.jsx]: Even better is: `sortBy(col) {` yes, with es2015 we don't need the function keyword when adding functions to dictionaries..

Comment (ckousik) [inline task-graph-inspector/taskgraphinspector.jsx]: yeah, but I wanted to keep it consistent with the way all the other functions are written

**Comment by eliperelman** [comment]
This needs a rebase and for review comments to be addressed. Closing until that is done.

**Pull request closed without merging**